module_auto_update/models/module.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2017 LasLabs Inc.
+# Copyright 2018 ACSONE SA/NV.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+import json
+import logging
+import os
+
+from odoo import _, api, exceptions, models, tools
+from odoo.modules.module import get_module_path
+
+from ..addon_hash import addon_hash
+
+PARAM_INSTALLED_CHECKSUMS = "module_auto_update.installed_checksums"
+PARAM_EXCLUDE_PATTERNS = "module_auto_update.exclude_patterns"
+DEFAULT_EXCLUDE_PATTERNS = "*.pyc,*.pyo,i18n/*.pot,i18n_extra/*.pot,static/*"
```

Comment:
Please align with https://github.com/acsone/click-odoo-contrib/blob/master/click_odoo_contrib/update.py#L28. Also update CONFIGURE.md. Maybe adjust the related test if necessary.
/